src/pymodaq_plugins_newport/daq_move_plugins/daq_move_Newport_Picomotor8742.py

```diff
@@ -0,0 +1,141 @@
+from pymodaq.control_modules.move_utility_classes import DAQ_Move_base, comon_parameters_fun, main  # common set of parameters for all actuators
+from pymodaq.utils.daq_utils import ThreadCommand # object used to send info back to the main thread
+from pymodaq.utils.parameter import Parameter
+from pylablib.devices import Newport
+
+
+
+class DAQ_Move_Newport_Picomotor8742(DAQ_Move_base):
+    """Plugin for the Picomotor 8742.
+
+    Attributes:
+    -----------
+    controller: Newport.Picomotor8742
+        The particular object that allow the communication with the hardware, from pylablib
+
+    """
+    _controller_units = ''
```

Comment:
add:
data_actuator_type = DataActuatorType.DataActuator

see discussion below for its implication